build/azure-pipelines.yml

```diff
@@ -22,8 +22,111 @@ trigger:
     include:
       - main
 
-pool:
-  vmImage: windows-2022
+variables:
+- template: variables.yml
 
-steps:
-- script: echo Hello World!
+stages:
+#-if false
```

Comment:
# Do we want to keep all the #-if false?

Reply:
Yes, it's used for code where we use it for the template but not for a generated application, so this code will be remove when you create a new application using the template.